build_sdk_windows_qt5_MSVC.bat

```diff
@@ -288,18 +288,13 @@ echo ####################################
 echo ###       ERROR occured          ###
 echo ####################################
 cd ..
-if '%WORKSPACE%'=='' (
-pause
-)
-rem exit 1
+exit 1
 
 :QUIT
 echo ************************************
 echo ***       SUCCESS finish         ***
 echo ************************************
 cd ..
 echo SDK installed in: %DLT_VIEWER_SDK_DIR%
-if '%WORKSPACE%'=='' (
-pause
-)
-rem exit 0
```

Comment:
Can we add here, like in the DLT Parser script?

set /p name= Continue

Then i can run it also on my desktop and see the result.